### ISSUE TITLE
Cross Entropy

### ISSUE THREAD

**Issue by jdavidd** [Jeetu95/Brain-Tumor-Segmentation#2]
Hello, Im trying to train on your netowork. With my training function, augmentation I only obtain 45% IOU accuracy with dice loss after 100 epochs, but learned very hard, small updates in accuracy. I try to use Crossentropy, but loss is constant, so I verify gradients and they 0, or almost 0. Do you have any ideas?

**Comment by Jeetu95** [comment]
When I was first using Pytorch and was training on a GPU I faced a similar problem of constant loss. Later on I found that just the model is put on GPU you need to put the loss function also on GPU.
Something like this
`model.to(device)`
`loss.to(device)`

Maybe this will fix your problem.
